tests/packages/sub_package/kitty/speak/hello.py

```diff
@@ -17,9 +17,13 @@
 #     See the License for the specific language governing permissions and
 #     limitations under the License.
 #
-def speak():
-    print( "hello kitty" )
 
 # Test Issue#115, in recursing modules, this was misbehaving.
 import types
+
+
+def speak():
```

Comment:
I am a bit unsure about whether this change makes the test no longer test what it is supposed to.

Reply:
Nah it's ok, what matters is that the type check with `==` is working fine. I tacked this on here apparently, to get this checked in module module, to not need another test, but the import does not matter where it is.